src/components/aktivitetskrav/vurdering/AvventAktivitetskravSkjema.tsx

```diff
@@ -42,36 +62,51 @@ export const AvventAktivitetskravSkjema = ({
       onSuccess: () => setModalOpen(false),
     });
   };
-  const { validateArsakerField, validateBeskrivelseField } =
-    useAktivitetskravVurderingSkjema();
-
-  const validate = (values: Partial<AvventAktivitetskravSkjemaValues>) => ({
-    ...validateArsakerField(values.arsaker),
-    ...validateBeskrivelseField(values.beskrivelse, true),
-  });
 
   return (
-    <Form onSubmit={submit} validate={validate}>
-      {({ handleSubmit }) => (
-        <form onSubmit={handleSubmit}>
-          <SkjemaHeading
-            title={texts.title}
-            subtitles={[texts.subtitle1, texts.subtitle2]}
-          />
-          <SkjemaFieldContainer>
-            <AvventArsakerCheckboxGruppe />
-            <VurderAktivitetskravBeskrivelse label={texts.beskrivelseLabel} />
-            <AvventFristDato />
-          </SkjemaFieldContainer>
-          {vurderAktivitetskrav.isError && (
-            <SkjemaInnsendingFeil error={vurderAktivitetskrav.error} />
-          )}
-          <LagreAvbrytButtonRow
-            isSubmitting={vurderAktivitetskrav.isLoading}
-            handleClose={() => setModalOpen(false)}
+    <FormProvider {...methods}>
```

Comment:
Måtte wrappe i `FormProvider` for å kunne bruke `useController` fra `<AvventFristDato>`